src/providers/generic/game/platform_interceptor/PlatformInterceptorImpl.ts

```diff
@@ -79,6 +80,13 @@ const RUNNERS: RunnerType = {
           "darwin": new DirectGameRunner(),
       }
     },
+    [StorePlatform.PC_GAME_PASS]: {
+        [PackageLoader.BEPINEX]: {
+            "win32": new DirectGameRunner(),
```

Comment:
I don't think these should be calling DirectGameRunner as we want to execute `gamelaunchhelper.exe` right?

A separate runner would be better `XboxGamePassGameRunner` (probably?) which instead of calling the first found executable in the GameManager.ts executables list (like DirectGameRunner does), it'd instead look for and call the `gamelaunchhelper` executable.

We also wouldn't need to declare the other executable names now. Although this is all worth making sure that args are passed through to the game when calling that executable.

Also a minor thing but since game pass isn't available on neither Linux nor MacOS, I'd say you don't need them defined. We can add that at a later date if necessary.